README.md

```diff
@@ -1,7 +1,7 @@
 # Git Queue
 
 <!-- markdownlint-disable-next-line MD013 -->
-[![Check dist/](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/check-dist.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/check-dist.yml) [![MegaLinter](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/mega-linter.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/mega-linter.yml) [![Test](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/test.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/test.yml) [![Test build](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/test-build.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/test-build.yml)
+[![Check dist/](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/check-dist.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/check-dist.yml) [![MegaLinter](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/mega-linter.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/mega-linter.yml) [![Test](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/test.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/test.yml) [![Test build](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/test-build.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/test-build.yml) [![CodeQL](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/codeql.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/codeql.yml) [![Deploy Documentation](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/deploy-documentation.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/deploy-documentation.yml) [![Publish GitHub Release](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/publish-github-release.yml/badge.svg)](https://github.com/Nautilus-Cyberneering/git-queue/actions/workflows/publish-github-release.yml)
```

Comment:
I checked the following paragraph below and I found one error of expression:

It should read "of one job at a time" not "of a job at the same time".